packages/wallet/test/hardware/ledger/LedgerKeyAgent.test.ts

```diff
@@ -298,6 +306,258 @@ describe('LedgerKeyAgent', () => {
+          const credential = getStakeCredential(
+            Cardano.RewardAccount('stake_test1up7pvfq8zn4quy45r2g572290p9vf99mr9tn7r9xrgy2l2qdsf58d')
+          );
+          const txBuilder = wallet.createTxBuilder();
+          txBuilder.partialTxBody.certificates = [
+            {
+              __typename: Cardano.CertificateType.AuthorizeCommitteeHot,
+              coldCredential: credential,
+              hotCredential: credential
+            }
+          ];
+          const tx = txBuilder
+            .addOutput(
+              txBuilder.buildOutput().address(outputs[0].address).coin(BigInt(outputs[0].value.coins)).toTxOut()
+            )
+            .build();
+
+          await expect(tx.sign()).rejects.toThrow(
+            InvalidDataReason.SIGN_MODE_ORDINARY__CERTIFICATE_COMMITTEE_COLD_CREDENTIAL_ONLY_AS_PATH
+          );
+        });
+
+        it('can sign a transaction with ResignCommitteeCold certs', async () => {
+          const credential = getStakeCredential((await firstValueFrom(wallet.delegation.rewardAccounts$))?.[0].address);
+          const txBuilder = wallet.createTxBuilder();
+          txBuilder.partialTxBody.certificates = [
+            {
+              __typename: Cardano.CertificateType.ResignCommitteeCold,
+              anchor: null,
+              coldCredential: credential
+            }
+          ];
+          const tx = txBuilder
+            .addOutput(
+              txBuilder.buildOutput().address(outputs[0].address).coin(BigInt(outputs[0].value.coins)).toTxOut()
+            )
+            .build();
+
+          // expect(await tx.sign()).toBeTruthy();
+          // TODO: should be able to sign?
+          await expect(tx.sign()).rejects.toThrow(InvalidDataReason.LEDGER_POLICY);
+        });
+
+        it('would throw while trying to sign a transaction with ResignCommitteeCold certs', async () => {
+          const credential = getStakeCredential(
+            Cardano.RewardAccount('stake_test1up7pvfq8zn4quy45r2g572290p9vf99mr9tn7r9xrgy2l2qdsf58d')
+          );
+          const txBuilder = wallet.createTxBuilder();
+          txBuilder.partialTxBody.certificates = [
+            {
+              __typename: Cardano.CertificateType.ResignCommitteeCold,
+              anchor: null,
+              coldCredential: credential
+            }
+          ];
+          const tx = txBuilder
+            .addOutput(
+              txBuilder.buildOutput().address(outputs[0].address).coin(BigInt(outputs[0].value.coins)).toTxOut()
+            )
+            .build();
+
+          await expect(tx.sign()).rejects.toThrow(
+            InvalidDataReason.SIGN_MODE_ORDINARY__CERTIFICATE_COMMITTEE_COLD_CREDENTIAL_ONLY_AS_PATH
+          );
+        });
+
+        it('can sign a transaction with RegisterDelegateRepresentative certs', async () => {
+          const credential = getStakeCredential((await firstValueFrom(wallet.delegation.rewardAccounts$))?.[0].address);
```

Comment:
The credential you are looking for is the drepKey. Please see [this example](https://github.com/input-output-hk/cardano-js-sdk/blob/master/packages/key-management/test/util/ownSignaturePaths.test.ts#L537)

Reply:
resolved